scripts/globals/moghouse.lua

```diff
@@ -166,6 +166,25 @@ xi.moghouse.moogleTrade = function(player, npc, trade)
                 player:messageSpecial(zones[player:getZoneID()].text.MOG_LOCKER_OFFSET + 2, xi.moghouse.getMogLockerExpiryTimestamp(player))
             end
         end
+
+        local eggComponents =
+        {
+            xi.items.EGG_LOCKER,
+            xi.items.EGG_TABLE,
+            xi.items.EGG_STOOL,
+            xi.items.EGG_LANTERN,
+        }
+
+        if npcUtil.tradeHasExactly(trade, eggComponents) then
```

Comment:
@claywar I don't remember which pairs of npcUtil trade stuff and player:confirmTrade etc. all go together (RE: Zaldon, etc.). Did we already talk about a CI check for counting/validating pairs of these calls in files?